lib/bolt/cli.rb

```diff
@@ -827,5 +841,11 @@ def config_loaded
       MSG
       @logger.debug(msg)
     end
+
+    # Gem installs include the aggregate, canary, and puppetdb_fact modules, while
+    # package installs include modules listed in the Bolt repo Puppetfile
+    def gem_install?
```

Comment:
This might better be named `incomplete_install?`, since it will also be the case for a fresh clone of Bolt.

Reply:
Updated!